projects/plugins/jetpack/_inc/client/at-a-glance/stats/index.jsx

```diff
@@ -97,12 +98,13 @@ export class DashStats extends Component {
 				nestedValue: null,
 				className: 'statsChartbar',
 				data: {
-					link: isOdysseyStatsEnabled
-						? `${ props.siteAdminUrl }admin.php?page=stats#!/stats/day/${ props.siteRawUrl }?startDate=${ date }`
-						: getRedirectUrl( `calypso-stats-${ unit }`, {
-								site: props.siteRawUrl,
-								query: `startDate=${ date }`,
-						  } ),
+					link:
+						isOdysseyStatsEnabled && ! isAtomicSite
```

Comment:
Wouldn't values `isOdysseyStatsEnabled` and `isAtomicSite` evaluate to truthy since they're selector functions? This would lead to this conditional always evaluating as false.

Reply:
Created a PR to address this.

https://github.com/Automattic/jetpack/pull/30341